cmd/commands/errors.go

```diff
@@ -0,0 +1,20 @@
+package commands
+
+import (
+	"errors"
+	"strings"
+)
+
+func MergeErrors(input []error) error {
+	mergedMessage := strings.Builder{}
+	for _, err := range input {
+		if err != nil {
+			mergedMessage.WriteString(err.Error() + "\n")
```

Comment:
not sure how I feel about new lines in the error message text when it's not formatted in a meaningful way.

Reply:
Shall I introduce a clearer demarcation or do you have something else in mind?

Reply:
Or shall the client be responsible for providing a delimiter? WDYT?